src/System.Drawing.Common/tests/Imaging/ImageAttributesTests.cs

```diff
@@ -1223,18 +1222,6 @@ public void SetOutputChannelColorProfile_InvalidPath_ThrowsOutOfMemoryException(
             }
         }
 
-        [ActiveIssue(22309)]
-        [ConditionalFact(Helpers.GdiplusIsAvailable)]
-        public void SetOutputChannelColorProfile_InvalidPath_ThrowsPathTooLongException()
```

Comment:
If it makes sense we can keep this test if we change 
`string fileNameTooLong = new string('a', 261);` to `string fileNameTooLong = new string('a', short.MaxValue);`